specification/columns/skupriceid.md

```diff
@@ -1,15 +1,26 @@
 
 The SkuPriceId column adheres to the following requirements:
 
-* SkuPriceId MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider publishes a SKU price list and MUST be of type String.
-* SkuPriceId MUST define a single unit price used for calculating the charge.
-* [ListUnitPrice](#listunitprice) MUST be associated with the SkuPriceId in the provider published price list.
-* SkuPriceId MUST NOT be null when [ChargeClass](#chargeclass) is not "Correction" and [ChargeCategory](#chargecategory) is "Usage" or "Purchase", MUST be null when ChargeCategory is "Tax", and MAY be null for all other combinations of ChargeClass and ChargeCategory.
-* A given value of SkuPriceId MUST be associated with one and only one [SkuId](#skuid), except in cases of [commitment discount flexibility](#glossary:commitment-discount-flexibility).
-* If a provider does not have a SkuPriceId and wants to include information in columns linked to SkuPriceId such as ListUnitPrice or [SkuPriceDetails](#skupricedetails), the SkuId MAY be used in the SkuPriceId column as long as it adheres to the above conditions.
+* SkuPriceId MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports unit pricing concepts and publishes price lists, publicly or as part of contracting.
+* SkuPriceId MUST be of type String.
+* SkuPriceId MUST conform to [String Handling](#stringhandling) requirements.
+* SkuPriceId nullability is defined as follows:
+  * SkuPriceId MUST be null when [ChargeCategory](#chargecategory) is "Tax".
+  * SkuPriceId MUST NOT be null when ChargeCategory is "Usage" or "Purchase" and [ChargeClass](#chargeclass) is not "Correction".
+  * SkuPriceId MAY be null in all other cases.
+* When SkuPriceId is not null, SkuPriceId adheres to the following additional requirements:
+  * SkuPriceId MUST have one and only one parent [SkuId](#skuid), except in cases of [commitment discount flexibility](#glossary:commitment-discount-flexibility).
+  * SkuPriceId MUST remain consistent over time.
+  * SkuPriceId MUST remain consistent across billing accounts within a provider.
+  * SkuPriceId MAY equal SkuId.
+  * SkuPriceId MUST be associated with a given [ResourceId](#resourceid) or [ServiceName](#servicename) used or purchased.
```

Comment:
And a purchase too ...

The following block demonstrates that the "SKU" is composite including the purchase SKU (`BCRM4X3Z935FE9QD`) and the resource SKU (`24EGBYAZB5NV9PSA`) that together produces the SKUPriceID of:
```
"discountedRate" : {
  "price" : "5.09715",
  "currency" : "USD"
},
```

<img width="673" alt="image" src="https://github.com/user-attachments/assets/aeb0c1f6-1750-432a-b3cb-e0fff9b36a1c" />

Reply:
@cnharris10 Resolving this since it seems like it's covered already. Feel free to unresolve if you feel anything is missing. Maybe leave a suggestion so we know what part should change.